uropa/visualization.py

```diff
@@ -26,29 +32,73 @@ def distribution_plot(table, var, kind):
     """
     pass
 
-
-def count_plot(table, var, kind):
+def count_plot(table, var="feature", kind="pie", title=None, title_size=20, path=None, dpi=300.0, label_rot=45):

```

Comment:
xaxis-tick-labels cut off when saving

![grafik](https://user-images.githubusercontent.com/27274175/195859666-95328587-baf9-44c6-bfc7-e877885c5f59.png)

Reply:
Also fixed. The axes are now saved fully when I test it locally.